src/rgw/rgw_op.cc

```diff
@@ -949,6 +950,7 @@ void RGWGetObj::execute()
 
   perfcounter->tinc(l_rgw_get_lat,
                    (ceph_clock_now(s->cct) - start_time));
+  get_lat_centile->insert(*read_op.params.obj_size, (ceph_clock_now(s->cct) - start_time).to_msec());
```

Comment:
you should follow the same coding style of the file to have this merged upstream.